docs/instrumentation.md

```diff
@@ -292,4 +292,19 @@ tpm_tis_spi.ko: tpm_tis_spi_probe()
 tpm_tis_spi.ko: tpm_tis_spi_driver_probe()
 tpm.ko: tpm_dev_release()
 tpm.ko: tpm_dev_release()
+
+```
+
+### Whole logs from the test procedure
+These logs will be created after running the python get_call_stack.py and 
+pull_logs.sh, according to the chapter above. Logs will be displayed in the 
+terminal window by default. </br>
+```concole
+$ ./pull_logs.sh
+```
+[Log file with  or without TPM module plugged in](./logs_tpm_module/log_with_tpm_pull_logs.txt)</br>
+```concole
+$ python get_call_stack.py 
 ```
+[Log file with TPM module plugged in](./logs_tpm_module/log_with_tpm_get_call_stack.txt)</br>
+[Log file without TPM](./logs_tpm_module/log_without_tpm_get_call_stack.txt)</br>
```

Comment:
remove html tags

Reply:
HTML tags removed in commit `0acd479aaffb4ea534d45381fa4415344b7cced6`